test/jdk/ProblemList.txt

```diff
@@ -796,6 +796,8 @@ com/sun/jdi/AfterThreadDeathTest.java                           8232839 linux-al
 
 # jdk_util
 
+java/util/Locale/LocaleProvidersRun.java                        8261919 generic-all
```

Comment:
This only needs to be ProblemListed on linux-x64.

Reply:
It's not platform specific, but specific to the environment that is usage tracker enabled. Thus I put it as 'generic'.